integr8ly/manifests
# add the cloud-resource-operator and bump the integreatly version

## Motivation
Add the cloud-resource-operator to the integreatly operator, needed for HA, 

## What
add the cloud-resource-operator and bump the integreatly version 

## Verification Steps
1. Push the changes above to your app registry
 - Run `make push/all`
 - **Verify**: cloud-resoure (0.1.0) and Integreatly Operator (1.7.6) should be available in your app registry
2. Create an `OperatorSource` with your app registry
e.g.
```yaml
apiVersion: operators.coreos.com/v1
kind: OperatorSource
metadata:
  finalizers:
    - finalizer.operatorsources.operators.coreos.com
  name: integreatly-operators
  namespace: openshift-marketplace
spec:
  authorizationToken: {}
  displayName: Integreatly Operators austincunningham
  endpoint: 'https://quay.io/cnr'
  publisher: austincunningham Publisher
  registryNamespace: austincunningham
  type: appregistry
```
 - **Verify**: CRO operator should be deployed
## Checklist:
